### PR TITLE
Add birthday celebration and recap blog

### PR DESCRIPTION
**Pull request by bitsondatadev**
Birthday recap blog

### CODE REVIEW

File: _posts/2022-09-02-tenth-birthday-celebration-recap.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
+---
+layout: post
+title: "Trino's tenth birthday celebration recap"
+author: "Brian Olsen"
+excerpt_separator: <!--more-->
+---
+
+What an exciting month we had in August! August marked the ten-year birthday of
```
Comment (mosabua) [inline _posts/2022-09-02-tenth-birthday-celebration-recap.md:8]: Too many `!` in this paragraph .. one should be enough

File: _posts/2022-09-02-tenth-birthday-celebration-recap.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+in one of the largest amount of page views on the Trino website in a given day
+(>25k views).
+
+![](/assets/blog/trino-tenth-birthday/hn-top.png)
+![](/assets/blog/trino-tenth-birthday/hn-9.png)
+
+## Trino ten-year timeline video
+
+Another way we celebrated was creating an epic ten-year montage video that
+chronicled the incredible journey starting with the Presto project's humble
```
Comment (mosabua) [inline _posts/2022-09-02-tenth-birthday-celebration-recap.md:37]: change sentence to present tense.. the video still exists

video that chronicles the incredible journey from the humble beginnings as Presto to the success that Trino is today.

File: _posts/2022-09-02-tenth-birthday-celebration-recap.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+<iframe src="https://www.youtube.com/embed/6TFLKcF24HM?clip=UgkxQcokpdgPjiuMKMC5-3HwHvlbmZjxAvxe&amp;clipt=EIv7YRiixmQ" width="800" height="500" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px;
+margin-bottom:5px; max-width: 100%;" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
+</iframe>
+
+Why table functions are so incredibly powerful.
+</div>
+</div>
+If you want to learn more about polymorphic table functions, check out the
+recent [Trino Community Broadcast episode]({{site.url}}/episodes/38.html) that
+covers the potential of these functions in much more detail!
```
Comment (mosabua) [inline _posts/2022-09-02-tenth-birthday-celebration-recap.md:137]: Change ! to .

File: _posts/2022-09-02-tenth-birthday-celebration-recap.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+
+### The early days of Presto and Trino
```
Comment (mosabua) [inline _posts/2022-09-02-tenth-birthday-celebration-recap.md:139]: Empty line after title

File: _posts/2022-09-02-tenth-birthday-celebration-recap.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+and building a distributed query engine from scratch. Some of the discussions
+were interesting while others were downright hilarious.
+
+<div class="three-column-grid">
+<div class="three-column-grid-item">
+<iframe src="https://www.youtube.com/embed/6TFLKcF24HM?clip=UgkxGjPYZRx8rhtAndyho7AZgsM4e9wG9Jt4&amp;clipt=EPbymQEYnN-cAQ" width="800" height="500" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px;
+margin-bottom:5px; max-width: 100%;" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
+</iframe>
+
+Step 1: Look up a bunch of research papers to see how others are doing this 📑:
```
Comment (mosabua) [inline _posts/2022-09-02-tenth-birthday-celebration-recap.md:151]: weird character there..

File: _posts/2022-09-02-tenth-birthday-celebration-recap.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+</div>
+
+<div class="three-column-grid-item">
+<iframe src="https://www.youtube.com/embed/6TFLKcF24HM?clip=Ugkx6Hqe5iglsTgrR9hVo9U3ITi8LSxxMu4U&amp;clipt=EN_2pAEYpMSoAQ" width="800" height="500" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px;
+margin-bottom:5px; max-width: 100%;" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="">
+</iframe>
+
+Site note: Papers tend to be highly aspirational and skip important fundamentals:
```
Comment (mosabua) [inline _posts/2022-09-02-tenth-birthday-celebration-recap.md:159]: Side note:

Comment (mosabua) [reply _posts/2022-09-02-tenth-birthday-celebration-recap.md:159]: also why does this sentence end with :

**Review by mosabua — CHANGES_REQUESTED** [review]
Lets chat about the layouting .. somehow weird where the div ends and : are placed .. seems to make no sense

File: _posts/2022-09-02-tenth-birthday-celebration-recap.md (hunk 5 above)
Comment (bitsondatadev) [reply _posts/2022-09-02-tenth-birthday-celebration-recap.md:151]: that's a files emoji

**Comment by mosabua** [comment]
General comment .. the way the post looks now is bad since it has the same image for each video. Either we make it so the previews are different or maybe easier .. just get rid of the funky layout and have a list of texts you have now with a "Play video" button or so.

**Review by mosabua — CHANGES_REQUESTED** [review]
Lets chat how to best proceed here..

File: _posts/2022-09-08-tenth-birthday-celebration-recap.md [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+restart Trino. With dynamic catalogs, you can add and remove these catalogs at
+runtime with no restart required. There is still no guarantee of exactly when
+this feature would arrive, but some of the foundations are currently being 
+added. <a href="https://www.youtube.com/clip/UgkxkYmwM6gmw9-GceMUb5IxqIKm0qNXt3fY" target="_blank"> 
+<i class="fab fa-youtube" style="color: red;"></i> Dain dives into this a bit
+more in this clip</a>
+
+### Vectorization and performance
+
+As more marketing around vectorized databases have come up recently many have
```
Comment (mosabua) [inline _posts/2022-09-08-tenth-birthday-celebration-recap.md:61]:
```suggestion
As more marketing around vectorized databases has come up recently many have
```

File: _posts/2022-09-08-tenth-birthday-celebration-recap.md [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+asked if Trino will be following the trend. This question comes up at an
+interesting time as [Trino supports and will soon require Java 17 to
```
Comment (mosabua) [inline _posts/2022-09-08-tenth-birthday-celebration-recap.md:63]:
```suggestion
interesting time as [Trino now requires Java 17 to 
```

File: _posts/2022-09-08-tenth-birthday-celebration-recap.md [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+recent [Trino Community Broadcast episode]({{site.url}}/episodes/38.html) that
+covers the potential of these functions in much more detail.
+
+### The early days of Presto and Trino
+
+We wanted to get some insight into what the early days of the project looked
+like, and how Martin, Dain, David, and Eric began the daunting task of designing
+and building a distributed query engine from scratch. Some of the discussions
+were interesting while others were downright hilarious. Here are some steps you
+can take to write your own query engine...at least if you want to do it the way
```
Comment (mosabua) [inline _posts/2022-09-08-tenth-birthday-celebration-recap.md:109]:
```suggestion
can take to write your own query engine, at least if you want to do it the way
```

File: _posts/2022-09-08-tenth-birthday-celebration-recap.md [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+    * Side note: Papers tend to be highly aspirational and skip important fundamentals.
+      [<i class="fab fa-youtube" style="color: red;"></i>
+      Video](https://www.youtube.com/clip/Ugkx6Hqe5iglsTgrR9hVo9U3ITi8LSxxMu4U){:target="_blank"}
+1. Address the real challenges of making a query engine.
+  [<i class="fab fa-youtube" style="color: red;"></i>
+  Video](https://www.youtube.com/clip/Ugkx57PezuXyRWHrxxxoLaKni6jqFZ-StwY-){:target="_blank"}
+1. Take your initial version and just throw it away 😂🗑🚮.
+  [<i class="fab fa-youtube" style="color: red;"></i>
+  Video](https://www.youtube.com/clip/UgkxJz7zve36QJZZDdtC3S29vI-Ak1jRifAH){:target="_blank"}
+1. Expand outside the initial use cases at Facebook by learning from other
```
Comment (mosabua) [inline _posts/2022-09-08-tenth-birthday-celebration-recap.md:124]:
```suggestion
1. Expand outside the initial use cases by learning from other
```

File: _posts/2022-09-08-tenth-birthday-celebration-recap.md [hunk 11]
```diff
@@ -0,0 +1,157 @@
...
+  companies and building community 👥.
+  [<i class="fab fa-youtube" style="color: red;"></i>
+  Video](https://www.youtube.com/clip/UgkxQrBl0BzOrjvwDcEN4KAAyqehcRUc1tsf){:target="_blank"}
+1. Cause a [brownout](https://en.wikipedia.org/wiki/Brownout_(software_engineering))
+  on the Facebook network 📉.
+  [<i class="fab fa-youtube" style="color: red;"></i>
+  Video](https://www.youtube.com/clip/Ugkx6SyQTFgwX_kdeH018VGt2pMUbldvuKtC){:target="_blank"}
+
```
Comment (mosabua) [inline _posts/2022-09-08-tenth-birthday-celebration-recap.md:132]: remove empty line

File: _posts/2022-09-08-tenth-birthday-celebration-recap.md [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+1. Realize the system you replaced was actually faster in some cases, but
+  for all the wrong reasons ❌🙅.
+  [<i class="fab fa-youtube" style="color: red;"></i>
+  Video](https://www.youtube.com/clip/UgkxTqBY2nMAALn-OkglE5DT9dHlBuC18qf8){:target="_blank"}
+
+After a lot of the initial work was done, Martin, Dain, David, and Eric finally
```
Comment (mosabua) [inline _posts/2022-09-08-tenth-birthday-celebration-recap.md:138]: This sentence makes no sense.

File: _posts/2022-09-08-tenth-birthday-celebration-recap.md [hunk 13]
```diff
@@ -0,0 +1,157 @@
...
+paid off. Prest was deployed at Facebook and soon after open sourced. From here,
```
Comment (mosabua) [inline _posts/2022-09-08-tenth-birthday-celebration-recap.md:139]:
```suggestion
paid off. Presto was deployed at Facebook and soon after open sourced. From here,
```

File: _posts/2022-09-08-tenth-birthday-celebration-recap.md [hunk 14]
```diff
@@ -0,0 +1,157 @@
...
+independent of Facebook, the features took off even more. While everything may
+seem calculated in hindsight, it was a lot of hard work to grow the community
+and adoption around Presto and now Trino. The creators knew they were making a 
+project that would be utilized outside the walls of Facebook, but
+[<i class="fab fa-youtube" style="color: red;"></i>  they could never have 
+anticipated the sheer scale of adoption Trino would see](https://www.youtube.com/clip/Ugkxh2J-1bi1rUoBpuld_FAuXYZgz2bvqPPx){:target="_blank"}.
+
+## Conclusion
+
+We hoped you enjoyed all the fun we had celebrating these first ten years of the
```
Comment (mosabua) [inline _posts/2022-09-08-tenth-birthday-celebration-recap.md:150]:
```suggestion
We hope you enjoyed all the fun we had celebrating the first ten years of the
```

**Review by mosabua — CHANGES_REQUESTED** [review]
Just a few more minor things to address. Nearly ready.

File: _posts/2022-09-12-tenth-birthday-celebration-recap.md [hunk 15]
```diff
@@ -0,0 +1,157 @@
...
+The creators knew they were making a project that would be utilized outside the
+walls of Facebook, but
+[<i class="fab fa-youtube" style="color: red;"></i>  they could never have 
+anticipated the sheer scale of adoption Trino would see](https://www.youtube.com/clip/Ugkxh2J-1bi1rUoBpuld_FAuXYZgz2bvqPPx){:target="_blank"}.
+
+## Conclusion
+
+We hope you enjoyed all the fun we had celebrating these first ten years of the
+Trino project. We are thrilled to think of what the following decades will
+bring. We'd like to leave you with a closing though from Dain around what the
```
Comment (mosabua) [inline _posts/2022-09-12-tenth-birthday-celebration-recap.md:152]:
```suggestion
bring. We'd like to leave you with closing thoughts from Dain:
```

File: _posts/2022-09-12-tenth-birthday-celebration-recap.md [hunk 16]
```diff
@@ -0,0 +1,157 @@
...
+next ten years of Trino will look like:
```
Comment (mosabua) [inline _posts/2022-09-12-tenth-birthday-celebration-recap.md:153]: remove

**Review by mosabua — CHANGES_REQUESTED** [review]

**Review by mosabua — APPROVED** [review]
Good to go now as well! 💥

**Comment by bitsondatadev** [comment]
> Uff .. just noticed .. hn-9.png is not used .. either use or delete

Deleted and merging!

**Comment by mosabua** [comment]
Sweet..

**Pull request merged**